Day-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './Footer.css';
+
+
+export const Footer = (props) => {
+
+    let replyClickHandler = () => alert("Reply!");
+    let retweetClickHandler = () => alert("Retweet!");
+    let heartClickHandler = () => alert("Heart!");
+    let shareClickHandler = () => alert("Share!");
```

Comment:
let 🆚const

Also, alerts are evil, use something else.

Reply:
I know, I wanted to actually implement the correct functionality but I ran out of time. alerts were the first thing I thought of.